setup_promtail.yml

```diff
@@ -22,6 +21,10 @@
     - group_vars/{{ env }}.yml
 
   tasks:
+    - name: Set service_identifier
```

Comment:
is it because it was taking the previous service identifier defined in set fact of setup_validator.yml ?

Reply:
also please move the comment with it too